### PR TITLE
Documentation

### LINKED ISSUE #9: Add documentation
### Type of Issue

- [ ] Bug
- [ ] Feature Request
- [X] Documentation
- [ ] Other

### Description

One of the main reason for this langate rewrite is the need for documentation. We will have 2 docs, one technical documentation as a mdbook example in the [website backend](https://github.com/InsaLan/backend-insalan.fr/tree/dev/docs) and the user documentation in the insalan wiki.

CI also need to be auto-deploying doc.

### PR DESCRIPTION
**Pull request by pixup1**
## Description

This PR adds documentation base files in the form of an mdbook similar to the one in [the website backend](https://github.com/InsaLan/backend-insalan.fr/tree/dev/docs/manuel), using release tags to compile the mdbook to html with github Actions. 

If this is merged, we need to add the following line to [mappings.txt](https://github.com/InsaLan/infra-insalan.fr/blob/main/docs/mappings.txt) in the infra-insalan.fr repo:

`https://github.com/InsaLan/langate3000/releases/latest/download/manuel.tar.gz /srv/http/docs/langate`

The [deploy_docs.sh](https://github.com/InsaLan/infra-insalan.fr/blob/main/docs/deploy_docs.sh) script should copy the docs to [docs.insalan.fr](docs.insalan.fr) periodically.

Structure and content of the documentation is mostly placeholder.

## Checklist

- [N/A] I have tested the changes locally and they work as expected.
- [N/A] I have tested the responsiveness of the changes and they work as expected.
- [x] I have assigned the pull request to the appropriate reviewer(s).
- [x] I have added labels to the pull request, if necessary.

## Related Issues

Fix #9

### CODE REVIEW

File: docs/manuel/src/01-frontend/style.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Style
+
+Le thème de la langate est réalisé en CSS.
```
Comment (KwikKill) [inline docs/manuel/src/01-frontend/style.md:3]: One line, one error :wink: 
The front is made with tailwind without css classes.

**Review by KwikKill — APPROVED** [review]
Good start for documentation, LGTM !

File: docs/manuel/src/01-frontend/style.md (hunk 1 above)
Comment (pixup1) [reply docs/manuel/src/01-frontend/style.md:3]: umm... it's placeholder anyway 😃👍

**Review by Ecnama — APPROVED** [review]
LGTM, ig

**Pull request merged**